contentcuration/kolibri_public/utils/mapper.py

```diff
@@ -57,7 +67,19 @@ def run(self):
             annotate_label_bitmasks(self.mapped_root.get_descendants(include_self=True))
             # Rather than set the ancestors fields after mapping, like it is done in Kolibri
             # here we set it during mapping as we are already recursing through the tree.
-            set_channel_metadata_fields(self.mapped_channel.id, public=self.public)
+
+            set_channel_metadata_fields(
+                self.mapped_channel.id,
+                channel_version=self.channel_version,
```

Comment:
Seems like we don't need to pass the `channel_version` to this method, since the channel already has its version set when it is copied in the `_map_model` method. And since this is the only place where we use this property, seems like we don't even need to get this info in the class constructor